### PR TITLE
[WIP] Initial call button work

### PR DESCRIPTION
**Pull request by Pomax**
This PR hooks up the call button actually do something:
- the call button on any page that is not `call-now` redirects the user to the `call-now` page
- the call button on the `call-now` page contacts our API server on the `/api/call` route, with the user-supplied phone number and chosen locale as POST payload.

UI follows https://redpen.io/vn42f08978cb80c431 and https://redpen.io/qa4807e518ba2b3439

THIS PR IS A WORK IN PROGRESS, DO NOT MERGE YET.

### CODE REVIEW

**Comment by Pomax** [comment]
For phone number validation, http://formvalidation.io/examples/validating-international-phone-numbers/ might be useful.

File: routes/call.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+var fetch = require('node-fetch');
+var FormData = require('form-data');
+
+const CALL_POWER_URL = process.env.CALL_POWER_URL;
+const COPYRIGHT_CAMPAIGN_ID = process.env.COPYRIGHT_CAMPAIGN_ID;
+
+/**
+ * Resolve a locale copde back to a country code. For
```
Comment (ScottDowne) [inline routes/call.js]: code*

**Comment by ScottDowne** [comment]
No red flags so far.

I noticed a few console.logs, and a typo in a comment, but that's it.

There isn't much harm in merging this when you're ready. The whole repo is in a WIP state at the moment, so like, we cannot break anything.

**Comment by Pomax** [comment]
fair point, I have the console logs there mainly to track flow, they shouldn't interfere with a deploy but I can take them out if desired for merge purposes, then put them back in to continue work.

**Pull request merged**